docs/best_practices.md

```diff
@@ -177,14 +177,14 @@ We use [Jest](https://jestjs.io/) for our unit tests, and [Mocha](https://mochaj
 Some top-level notes:
 
 - ~~We use [`enzyme`](https://enzymejs.github.io/enzyme/)~~
-- As of 2021, we've begun using [@testing-library/react](https://testing-library.com/docs/) for our tests. (See [this doc](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find) for some common mistakes and best practices.)   
+- As of 2021, we've begun using [@testing-library/react](https://testing-library.com/docs/) for our tests. (See [this doc](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find) for some common mistakes and best practices.)
```

Comment:
Maybe removing the anchor from the link to scroll to the beginning instead of that section

Reply:
Ah, good call 👍

Reply:
(done: https://github.com/artsy/force/commit/e5ff5dddcd5b8108b1fa69ba688e39461e08a497)